tests/jahia-module/pom.xml

```diff
@@ -0,0 +1,89 @@
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+    As a special exception to the terms and conditions of version 2.0 of
+    the GPL (or any later version), you may redistribute this Program in connection
+    with Free/Libre and Open Source Software ("FLOSS") applications as described
+    in Jahia's FLOSS exception. You should have received a copy of the text
+    describing the FLOSS exception, and it is also available here:
+    http://www.jahia.com/license
+
+    Commercial and Supported Versions of the program (dual licensing):
+    alternatively, commercial and supported versions of the program may be used
+    in accordance with the terms and conditions contained in a separate
+    written agreement between you and Jahia Solutions Group SA.
+
+    If you are unsure which license is appropriate for your use,
+    please contact the sales department at sales@jahia.com.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>jahia-modules</artifactId>
+        <groupId>org.jahia.modules</groupId>
+        <version>8.1.1.0</version>
+        <relativePath />
+    </parent>
+    <groupId>org.jahia.test</groupId>
+    <artifactId>content-editor-test-module</artifactId>
+    <name>Test module for content-editor</name>
+    <version>4.4.0-SNAPSHOT</version>
+    <packaging>bundle</packaging>
+    <description>This is the custom module (text-field-initializer) for content-editor cypress tests.</description>
+
+    <repositories>
+        <repository>
+            <id>jahia-public</id>
+            <name>Jahia Public Repository</name>
+            <url>https://devtools.jahia.com/nexus/content/groups/public</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Jahia-Depends>default</Jahia-Depends>
+                        <Jahia-Signature>MCwCFAcSeckcHaD8fZd7cfvJhGJnACb+AhQwFP3aS3BfhY8IponyZsdRJsL+oA==</Jahia-Signature>
```

Comment:
I'm not sure that we need a signature for modules which are not under the groupId <groupId>org.jahia.modules</groupId>

Reply:
You are absolutely right, thanks, fixed